ethexe/contracts/test/Middleware.t.sol

```diff
@@ -5,19 +5,25 @@ import {EnumerableMap} from "@openzeppelin/contracts/utils/structs/EnumerableMap
 import {Time} from "@openzeppelin/contracts/utils/types/Time.sol";
 import {MessageHashUtils} from "@openzeppelin/contracts/utils/cryptography/MessageHashUtils.sol";
 import {Upgrades} from "openzeppelin-foundry-upgrades/Upgrades.sol";
-
 import {Test, console} from "forge-std/Test.sol";
+import "forge-std/Vm.sol";
```

Comment:
```suggestion
import {Vm} from "forge-std/Vm.sol";
```
iirc `vm.` should be available w/o import in tests

Reply:
done